openedx_events/tests/test_tooling.py

```diff
@@ -37,6 +53,14 @@ def setUp(self):
             data=self.data_attr,
         )
 
+        self.receiver_error = Exception("fake error")
+
+        def error_receiver(*args, **kwargs):
+            raise self.receiver_error
+
+        self.ok_receiver = Mock(return_value="success")
+        self.error_receiver = error_receiver
```

Comment:
Author note: These test receivers and `receivers_attached` could be used in other tests in this file, which is why they're up here. If you don't think we'll get around to using them in other tests, I can collapse them back down into the one place they're used.

Reply:
This looks really cool! And I see it can be used elsewhere if needed